le.go

```diff
@@ -77,6 +79,10 @@ func (logger *Logger) isOpenConnection() bool {
 		return false
 	}
 
+	if time.Now().After(logger.lastRefreshAt.Add(15 * time.Minute)) {
+		return false
```

Comment:
Shall we `logger.conn.Close()` here too?

Reply:
@Prendo93 Done